src/containers/Search.js

```diff
@@ -20,12 +20,14 @@ class search extends React.Component {
       todosPerPage: 6,
       order:'',
       filters:{org:[],team:[]},
+      isChecked: props.isChecked || true,
```

Comment:
When the app is at mobile size the filter menu appears to be open by default when I search. So I go Home page -> start search -> go to results page -> and the panel is already open by default blocking the results. It can be solved by setting this to `false`